.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Note: Should probably include information about setting up Prettier in setup documentation.

Reply:
@MJJBennett I've added a note for this in #14 which I'll be working on after this PR is merged!